README.md

```diff
@@ -1,4 +1,20 @@
 # AndroidDatabaseLibraryComparison
 A test between a few of the popular libraries running a speed test on how fast they load and save data.
 
-**Note** more to come soon about these with results and methods.
+## Benchmark Description
+
+There are two benchmarks.  The Simple trial uses a flat schema for an address book so each row is composed of name, address, city, state, and phone columns.  The Complex trial is hierarchical and has support for multiple address books where each address book has contacts and addresses.
```

Comment:
#req we should separate each trial and diagram the schema in a visual way or by listing each table and relationship. (UML?)